ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -167,6 +167,17 @@ export const useChatbot = () => {
           }
         },
       },
+      copy: {
+        onClick: () => {
+          if (message.actions) {
+            message.actions.copy.isDisabled = true;
```

Comment:
I think we do not need to disable the copy icon after a user clicked it once.  Would you remove this line?

Reply:
ack, updated and removed the `disabled` check.